Documentation/Makefile

```
@@ -45,7 +45,32 @@ copy-api:
 	@$(ECHO_GEN)_api
 	$(QUIET)cp -r ../api _api
 
-epub latex html: builder-image copy-api
+HELM_VALUES = helm-values.rst
+
+HELM_DOCS_VERSION := v1.3.0
+HELM_DOCS_SHA := 12f065115654c9a1d6819b938487d327f1fdb5c32bdc2a5c74afef22ed68f5ca
+HELM_DOCS_IMAGE := docker.io/jnorwood/helm-docs:$(HELM_DOCS_VERSION)@sha256:$(HELM_DOCS_SHA)
+HELM_DOCS_ROOT_PATH := /src
+HELM_DOCS_CHARTS_DIR := $(HELM_DOCS_ROOT_PATH)/install/kubernetes
+HELM_DOCS_OUTPUT_DIR := $(HELM_DOCS_ROOT_PATH)/Documentation
+HELM_DOCS := $(DOCKER_CTR) -v $(PWD)/..:$(HELM_DOCS_ROOT_PATH) $(HELM_DOCS_IMAGE)
+
+M2R_VERSION := 0.2.1
+M2R_SHA := a3dc2c3fcdc31260dcb7cd42e39763281a6fdce4ca59b4c150fb3bfff5be9eb7
+M2R_IMAGE := docker.io/bmcustodio/m2r:$(M2R_VERSION)@sha256:$(M2R_SHA)
+M2R := $(DOCKER_CTR) $(M2R_IMAGE) m2r
+
+.PHONY: $(HELM_VALUES)
+$(HELM_VALUES): TMP_FILE := helm-values.tmp
+$(HELM_VALUES): AWK_FILE := helm-values.awk
+$(HELM_VALUES):
+	$(QUIET)touch $(TMP_FILE) $(AWK_FILE)
+	$(QUIET)$(HELM_DOCS) -d -c $(HELM_DOCS_CHARTS_DIR) -t $(HELM_DOCS_OUTPUT_DIR)/$(TMP_FILE).tmpl > $(TMP_FILE)
+	$(QUIET)awk -F'|' '{print "|"$$2"|"$$5"|"$$3"|"$$4"|"}' $(TMP_FILE) > $(AWK_FILE)
+	$(QUIET)$(M2R) --overwrite $(AWK_FILE)
+	$(QUIET)rm $(TMP_FILE) $(AWK_FILE)
```

```suggestion
	$(QUIET)$(RM) -- $(TMP_FILE) $(AWK_FILE)
```
Good practice: Makefile usually use `rm -f` for `$(RM)`, and the `--` prevents adding additional options by accident when expanding the variables.